### PR TITLE
Mapp Chosen Recipies in weekly overview

### LINKED ISSUE #55: Save Dates with Recipe in DB
- [x]  save dates in dbJSON
- [x]  save chosen Recipe for that chosen date in dbJSON
- [ ]  select a range for how long/how many items are getting saved

### PR DESCRIPTION
**Pull request by boldCM**
- displays chosen Recipies in start Page
- select and save a date for a chosen Recipe
- shows the chosen Date while initial loading

### CODE REVIEW

File: client/src/api/connectJSON.js [hunk 1]
```diff
@@ -42,3 +42,14 @@ export async function deleteRecipeFromWeek(Id) {
     method: "DELETE",
   });
 }
+
+export async function updateWeekByDate(date, Id) {
+  const saveDate = { date: date };
```
Comment (mariegross) [inline client/src/api/connectJSON.js]:
```suggestion
  const saveDate = { date };
```

If key and value are identical, you can use this short form.

File: client/src/components/WeekListItem/WeekListItem.js [hunk 2]
```diff
@@ -20,6 +22,9 @@ const StyledDatePicker = styled(DatePicker)`
   :focus {
     outline: none;
   }
+  ::placeholder {
```
Comment (mariegross) [inline client/src/components/WeekListItem/WeekListItem.js:25]:
```suggestion
  :placeholder {
```
Looks like there's one too many colons there :)

**Review by mariegross — APPROVED** [review]
Looks great, nice work! 👍 
Left only two little suggestions :)

File: client/src/api/connectJSON.js (hunk 1 above)
Comment (boldCM) [reply client/src/api/connectJSON.js]: oh thanks, didn't know that! 😍

File: client/src/components/WeekListItem/WeekListItem.js (hunk 2 above)
Comment (boldCM) [reply client/src/components/WeekListItem/WeekListItem.js:25]: I'll check on that one...
....as far as I know it has to be written with double ::

**Pull request merged**